parm/config/config.resources

```diff
@@ -137,9 +138,15 @@ elif [ $step = "anal" ]; then
       export npe_anal=84
       export npe_anal_gfs=84
     fi
-    if [[ "$machine" = "WCOSS_DELL_P3" ]]; then export nth_anal=7; fi
+    if [[ "$machine" = "WCOSS_DELL_P3" ]]; then 
+      export nth_anal=4
+      if [ "$QUEUE" = "dev2" -o "$QUEUE" = "devonprod2" -o "$QUEUE" = "devmax2" ]; then # WCOSS Dell 3.5
+        export nth_anal=5
+      fi
```

Comment:
Why is there a `QUEUE` dependence here?

Reply:
v16iasicris2 parallel used this rule when running with these QUEUE.

Reply:
I suggest remove the `QUEUE` logic and replace 4 with 5 on Line 141.